zkevm-circuits/src/evm_circuit/execution/extcodehash.rs

```diff
@@ -6,38 +6,31 @@ use crate::{
         table::{AccountFieldTag, CallContextFieldTag},
         util::{
             common_gadget::SameContextGadget,
-            constraint_builder::{ConstraintBuilder, StepStateTransition, Transition::Delta},
-            from_bytes, Cell, RandomLinearCombination, Word,
+            constraint_builder::{
+                ConstraintBuilder, ReversionInfo, StepStateTransition, Transition::Delta,
+            },
+            from_bytes,
+            math_gadget::BatchedIsZeroGadget,
+            Cell, RandomLinearCombination, Word,
         },
         witness::{Block, Call, ExecStep, Transaction},
     },
     util::Expr,
 };
-use eth_types::ToLittleEndian;
-use eth_types::{evm_types::GasCost, Field, ToAddress, ToScalar, U256};
-use ethers_core::utils::keccak256;
+use eth_types::{evm_types::GasCost, Field, ToAddress, ToScalar, EMPTY_HASH_LE, U256};
 use halo2_proofs::{circuit::Region, plonk::Error};
-use lazy_static::lazy_static;
-
-lazy_static! {
-    static ref EMPTY_CODE_HASH_LE_BYTES: [u8; 32] = U256::from(keccak256(&[])).to_le_bytes();
-}
```

Comment:
This is how I would define `EMPTY_HASH` but instead of here, probably in the keccak repo.
Maybe in `mock` crate in the case is only used for testing.

Reply:
Make sense to me, I have added it back in crate `keccak256` in [`6794b0f`](https://github.com/appliedzkp/zkevm-circuits/pull/278/commits/6794b0f4bec7c4bf9c5acfb3cf9f6a0cb2cbbf21)

Reply:
Thanks!!!